source/language/types.rst

```diff
@@ -171,6 +241,20 @@ be true and 0 will be false.
    // Assign a cast bit to a boolean
    my_bool = bool(my_bit);
 
+Needs clarification:
+--------------------
+Wouldn't a C-style cast
+-----------------------
+
+.. code-block:: c
+  my_bool = (bool) my_bit;
+
+be more consistent with C? The cast example above is more like an
+-----------------------------------------------------------------
+explicit construction / conversion from C++.
+--------------------------------------------
```

Comment:
I definitely prefer the conversion syntax we have proposed. We will need a more functional form to support some of the things above where we need to define how a value is extended or contracted into smaller/larger types.